packages/core/strapi/lib/commands/builders/tasks/vite.js

```diff
@@ -0,0 +1,137 @@
+'use strict';
+
+const path = require('path');
+const { build, createLogger } = require('vite');
+const react = require('@vitejs/plugin-react');
+const ora = require('ora');
+const chalk = require('chalk');
+
+/**
+ * @internal
+ *
+ * @type {(ctx: import('../packages').BuildContext, task: ViteTask) => import('vite').UserConfig}
+ */
+const resolveViteConfig = (ctx, task) => {
+  const { cwd, distPath, targets, external, extMap, pkg } = ctx;
+  const { entries, format, output, runtime } = task;
+  const outputExt = extMap[pkg.type || 'commonjs'][format];
+  const outDir = path.relative(cwd, distPath);
+
+  const customLogger = createLogger();
+  customLogger.warn = (msg) => ctx.logger.warn(msg);
+  customLogger.warnOnce = (msg) => ctx.logger.warn(msg);
+  customLogger.error = (msg) => ctx.logger.error(msg);
+
+  /**
+   * @type {import('vite').InlineConfig}
```

Comment:
I'm not sure we can, `defineConfig` expects `UserConfig` but we want to use `InlineConfig` because we want to set `configFile` to false so we _do not_ look for a vite config

Reply:
Ho ok I didn't notice it wasn't the same 👍